main/src/ca/uwaterloo/flix/api/lsp/provider/FindReferencesProvider.scala

```diff
@@ -75,26 +75,30 @@ object FindReferencesProvider {
   }
 
   private def findClassUses(sym: Symbol.ClassSym)(implicit index: Index, root: Root): JObject = {
+    val clazz = Location.from(sym.loc)
     val uses = index.usesOf(sym)
-    val locs = uses.toList.map(Location.from)
+    val locs = clazz :: uses.toList.map(Location.from)
     ("status" -> "success") ~ ("result" -> locs.map(_.toJSON))
```

Comment:
Suggestion: Let us use the same two local variable names in all of these. Specifically i propose:

- "defSite"
- "useSites"

and then we always write `defSite :: useSites`.

Makes sense?

Reply:
sure totally :)

Reply:
Very good. I think we should rename the functions like you did before too. E.g. `findClassUses` becomes `findClassReferences`. Agree?

Reply:
Oh yes you are completely true, I am sorry for that i completely forgot

Reply:
I updated as suggested, I have also fixed the references of tags :)